realm/realm-library/src/objectServer/java/io/realm/SyncUser.java

```diff
@@ -62,7 +61,26 @@
 @Beta
 public class SyncUser {
 
-    private SyncConfiguration managementRealmConfig;
+    private static class ManagementConfig {
+        private SyncConfiguration managementRealmConfig;
+
+        synchronized SyncConfiguration initAndGetManagementRealmConfig(
+                ObjectServerUser syncUser, SyncUser user)
+        {
```

Comment:
`{` should be on the same line 68. for java, we don't use new line for function `{`